src/LinuxTracing/TracerImpl.cpp

```diff
@@ -718,6 +734,38 @@ static std::vector<ThreadName> RetrieveInitialThreadNamesSystemWide(uint64_t ini
   return thread_names;
 }
 
+// Returns a vector of pairs mapping the tid in the target process namespace to the corresponding
+// tid in the root namespace.
+static std::vector<std::pair<pid_t, pid_t>> RetrieveInitialTidMapping(pid_t target_pid) {
```

Comment:
Again, this name is too generic, "tid mapping" can mean too many things. See the other comment.